conan/internal/api/uploader.py

```diff
@@ -271,16 +271,17 @@ def gzopen_without_timestamps(name, fileobj, compresslevel=None):
     return t
 
 
-def compress_files(files, name, dest_dir, compresslevel=None, ref=None):
+def compress_files(files, name, dest_dir, compresslevel=None, ref=None, recursive=False):
     t1 = time.time()
     # FIXME, better write to disk sequentially and not keep tgz contents in memory
     tgz_path = os.path.join(dest_dir, name)
-    ConanOutput(scope=str(ref)).info(f"Compressing {name}")
+    if ref:
```

Comment:
Why this ``ref`` check?
Don't we want the message of this being compressed always?

Reply:
Yes. If not, it will add messages like this one:
```
None: Compressing conan_cache_save.tgz
```
Which is not intended. And this will be added to `conan cache save` output.